docs/Growing.md

```diff
@@ -117,6 +117,132 @@ functionality and then moves on to producing code that passes these tests. There
 having good test coverage and code being designed for testability, which will in turn often prevent future changes from
 breaking old code. Much of mojo.js was developed using TDD.
 
+## Prototype
+
+An important difference between mojo.js and other web frameworks is that it can operate in two modes, both as a 
+full-fledged web framework in single-file mode, and as a micro web framework optimized for rapid prototyping.
+
+### Differences
+
+You likely know the feeling, you've got a really cool idea and want to try it as quickly as possible. That's exactly
+why mojo.js apps can be run as a single file (plus package.json for deps).
```

Comment:
The `...can be run as a single file (plus package.json for deps)` can probably be phrased better.

Reply:
Actually the original still sounds good here in context, `...that's exactly
why mojo.js applications don't need more than a single file.`